Source/MediaInfo/Image/File_Tiff.cpp

```diff
@@ -204,141 +204,165 @@ static const char* Tiff_ExtraSamples_ColorSpace(int32u ExtraSamples)
 void File_Tiff::Streams_Finish()
 {
     if (Retrieve_Const(Stream_General, 0, General_Format).empty()) {
-        Fill(Stream_General, 0, General_Format, "TIFF");
+        const auto FileExt{ Retrieve_Const(Stream_General, 0, General_FileExtension) };
+        if (FileExt == __T("arw") || FileExt == __T("ARW"))

```

Comment:
The format should not depend on the file name (sometimes we don't have it), isn't a way in the tags to catch the format?